website/Test/FluidOrg/Convolution.purs

```diff
@@ -18,6 +18,6 @@ testFig :: T.Page -> Aff Unit
 testFig page = do
    let figId = "fig"
    waitForFigure page (figId <> "-output")
-   let toggle = figId <> "-input"
-   clickToggle page toggle
-   waitFor (T.Selector ("div#" <> toggle)) page
+   let toggle = "-data-pane"
```

Comment:
I think we discussed moving this `-data-pane` suffix behaviour into `clickToggle`